lib/pipefy_message/consumer.rb

```diff
@@ -79,9 +79,9 @@ def process_message
 
           obj.perform(payload["Message"])
 
-          elapsed_time = (Time.now - start) * 1000.0
+          elapsed_time_ms = (start - Process.clock_gettime(Process::CLOCK_MONOTONIC)) * 1000
```

Comment:
Shouldn't this be Process.clock_gettime(Process::CLOCK_MONOTONIC) - start? I'm not familiar with this method, though.

Reply:
Also, from the documentation for that method, maybe we could call it like `Process.clock_gettime(Process::CLOCK_MONOTONIC, unit: :millisecond)` rather than multiplying it by 1000. I think having an explicit unit makes it more readable and less error-prone.

Reply:
Though I've just checked and it seems that setting a unit makes it return an integer; not sure if that's ideal.

Reply:
I tested here and in the way that you made @gabriel376 we'll have negative milliseconds.

@ferreira-mev I liked a lot your suggestion, we just have to use it in both variables (start and elapsed_time_ms). About the milliseconds in integer precision, I don’t think that is a problem by now

Reply:
Yes, good catch!